fastlane/lib/fastlane/actions/git_add.rb

```diff
@@ -76,12 +71,12 @@ def self.is_supported?(platform)
       def self.example_code
         [
           'git_add',
-          'git_add(path: "./version.txt")',
+          # 'git_add(path: "./version.txt")',
           'git_add(path: ["./version.txt", "./changelog.txt"])',
-          'git_add(path: "./Frameworks/*", shell_escape: false)',
+          # 'git_add(path: "./Frameworks/*", shell_escape: false)',
           'git_add(path: ["*.h", "*.m"], shell_escape: false)',
-          'git_add(path: "./Frameworks/*", shell_escape: false)',
-          'git_add(path: "*.txt", shell_escape: false)'
+          # 'git_add(path: "./Frameworks/*", shell_escape: false)',
+          # 'git_add(path: "*.txt", shell_escape: false)'
```

Comment:
👀  Force change, to make `test_sample_code` happy
https://github.com/fastlane/fastlane/blob/master/fastlane/actions/test_sample_code.rb 

<img width="636" alt="Screenshot 2021-06-13 at 17 10 38" src="https://user-images.githubusercontent.com/5364500/121813105-e5447280-cc6a-11eb-91e2-688a311bef15.png">